### ISSUE TITLE
Accessibility failures on Leaflet markers and popups

### ISSUE THREAD

**Issue by peteredwards** [uol-library/spacefinder-ui#19]
**WCAG Criterion 2.1.1 Keyboard (Level A)**

* Cannot always navigate to the close button on the map dot modals via keyboard. Particularly after a group of dots has been clicked on. 
* Unable to open grouped map dots via keyboard.

**WCAG Criterion 2.4.3 Focus Order (Level A)**

* Focus doesn't move to the map popups triggered by selecting the markers
* Cannot close the map popups via keyboard that open after selecting a map dot with keyboard
* Cannot determine whether focus is returned to the marker after the popup has closed

**Comment by peteredwards** [comment]
I've updated both Leaflet and the markerclusterer plugin as I noticed there was some movement in these projects on issues with keyboard access to clusters and moving focus to popups. I'll try to get these issues re-tested
